### ISSUE TITLE
Accessibility testing

### ISSUE THREAD

**Issue by klohman** [GSA/oasis-plus#163]
Before the new GSA SmartPay program site goes live, we need to do some manual & automated testing, to ensure the new program site is accessible:

 - [ ] Run lighthouse
 - [ ] 508 testing
> -  [AMP](https://insite.gsa.gov/employee-resources/information-technology/it-accessibility-section-508/accessibility-solutions/accessibility-management-platform-amp?check_logged_in=1)
> -  [WAVE evaluation tool](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)

- [ ] Keyboard navigation testing 
- [ ] Testing with 200% zoom
- [ ] Screen reader testing  
- [ ] Log any issues discovered

**Comment by weiwang-gsa** [comment]
@rosamundtgov For DPA page, there are same" OASIS+ Interact Community" text linked to slightly different URL, which cause accessibility complaints , one is https://buy.gsa.gov/interact/community/196/activity-feed, another one is https://buy.gsa.gov/interact/community/196, can we sync both using same link, e.g. both use https://buy.gsa.gov/interact/community/196/activity-feed?

**Comment by weiwang-gsa** [comment]
For Home page color contracts issue (AMP & WAVE), for "Find out what’s next" section, actually we are using a dark background image, but these automated tool are checking against background color (white) vs foreground text color( also white), I did manual color contrast ratio test for the foreground text vs background image, the color contrast ratio is good and passed the accessibility requirement, please see screen shot below. So this color contrast ratio issue on home page can be ignored. 



![Image](https://github.com/GSA/oasis-plus/assets/119954777/cd3929de-3c7a-4d82-9d06-24cede6c04c5)

**Comment by jfredrickson5** [comment]
How about adding pa11y to do a bit of automated testing?

**Comment by vickimcfadden** [comment]
Seems like a good idea to me @jfredrickson5 , @mark-meyer @weiwang-gsa ☝ what do you think?

**Comment by weiwang-gsa** [comment]
@vickimcfadden yes, we will add the pa11y for automated testing, just let you know that pa11y can only catch obvious accessibility issue, it can't catch intense accessibility issues like AMP does though, manual testings are still needed.

**Comment by rosamundtgov** [comment]
> [@rosamundtgov](https://github.com/rosamundtgov) For DPA page, there are same" OASIS+ Interact Community" text linked to slightly different URL, which cause accessibility complaints , one is https://buy.gsa.gov/interact/community/196/activity-feed, another one is https://buy.gsa.gov/interact/community/196, can we sync both using same link, e.g. both use https://buy.gsa.gov/interact/community/196/activity-feed?

Let's use https://buy.gsa.gov/interact/community/196 across the board. I'll make a note on another ticket to update this other places if needed.

**Comment by weiwang-gsa** [comment]
@vickimcfadden automated accessibility test pa11y is added to the project and code merged into main. @rosamundtgov the link update for "OASIS+ Interact Community" is completed and code also merged into main.

**Comment by weiwang-gsa** [comment]
Just confirmed with @klohman, the "OASIS+" as H1 header causing two H1 header on one page issue can be ignored for now until we get new OASIS+ logo to replace it. 

@rosamundtgov, for Contact page, the page title currently set to "Contact", which is too short and AMP doesn't like it, is it possible we change both page title and h1 header to "Contact OASIS+" or something longer that you think more appropriate?  I tagged you in the accessibility findings as well just in case you want to know what is the issue.

**Comment by mark-meyer** [comment]
For items under **3.Top Menu**

-  First item is only reproducible when running locally using Astro's dev server. It is caused by the tool bar astro inserts on the page in development mode
- The issue *"Ensure ARIA regions, landmarks and HTML sections are identifiable"* has been addressed by ensuring the Aria labels are unique.
- The issues around *"This A is focusable and has an aria-hidden attribute set to true"* is caused by USWDS javascript adding aria-hidden to main page elements when the side navigation is open. We are relying on USWDS for this — if this is an issue the ends up impacting users, we would need to address this with the USWDS team.

**Comment by klohman** [comment]
Keyboard testing complete and no issues found.

**Comment by klohman** [comment]
Also reviewed with 200% zoom on Chrome and saw no issues. @brentryanjohnson If you want to take a quick look as well.

**Comment by brentryanjohnson** [comment]
I tested the `main` branch at 200%...I didn't noticed any specific problems, other than the side (in-page) nav gets cropped out of the viewport, which likely isn't a huge deal.

As an aside, I did notice the button link on this page is broken. I suspect that's probably a known issue. (Also, these icons are 💯!)

![A list of key features with blue seal icons with a button at the bottom that reads Learn more about how oasis plus compares to oasis](https://github.com/GSA/oasis-plus/assets/32855580/de10c9cf-e719-4199-9969-ced4b076ba22)

**Comment by klohman** [comment]
@brentryanjohnson Thanks for the review. I added the broken link to #120

**Comment by vickimcfadden** [comment]
I think we have all of the issues identified in other cards and/or done. Moving this to done.
